integration-tests/e2e-runtime-only-backend/lib.rs

```diff
@@ -73,5 +73,22 @@ pub mod flipper {
 
             Ok(())
         }
+
+        /// Just instantiate a contract using non-default runtime.
+        #[ink_e2e::test(backend = "runtime-only", runtime = drink::runtime::MinimalRuntime)]
```

Comment:
Can't you do?
```suggestion
        #[ink_e2e::test(backend = "runtime-only", runtime = ::ink_e2e::MinimalRuntime)]
```

Reply:
oh sure, that's why I reexported it (but somehow I forgot about it here...); fixed